future-apps/payroll/contracts/PayrollKit.sol

```diff
@@ -69,76 +70,169 @@ contract PayrollKit is KitBase {
 
```

Comment:
(was not able to put this above)
`KitBase` is now published, so it can be imported instead of copied here again. See: 
https://github.com/aragon/dao-kits/issues/45
https://github.com/aragon/dao-kits/pull/46